cabal.project

```diff
@@ -319,92 +319,92 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: 34e380f54ed24772479763ba6d67f4893d82aac2
-  --sha256: 03gnxiq6rl6j1wnbw1nj5rix1iga406yyp1v9xxz4684qsxj5b24
+  tag: 46e9db4dd722ac87a3cf2ace70955a72b31fe22b
```

Comment:
I don't think we should depend on a revision that isn't on `master`.

Reply:
Ah sorry @deepfire, didn't see "[TEST]" in the title of this PR.

Reply:
That's not intended for merging yet, sure.

Reply:
I should have made it a Draft PR, but made a mistake.